_stories/molecules/Breadcrumbs/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { select, text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+import readme from './README.md';
+import Breadcrumbs from '../../../components/molecules/Breadcrumbs';
+
+const config = {
+    title: 'Home',
+    path: 'home',
+    subpaths: [
+        {
+            title: 'Main Category',
+            path: 'category',
+            subpaths: [
+                {
+                    title: 'Sub Category 1',
+                    path: 'subcategory-1'
+                },
+                {
+                    title: 'Sub Category 2',
+                    path: 'subcategory-2'
+                },
+                {
+                    title: 'Sub Category 3',
+                    path: 'subcategory-3'
+                }
+            ]
+        },
+        {
+            title: 'Products',
+            path: 'products',
+            subpaths: [
+                {
+                    path: ':id',
+                    subpaths: [
+                        {
+                            path: 'edit'
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            path: ':pageId'
+        }
+    ]
+};
+
+class BreadcrumbsStory extends Component {
```

Comment:
Can we improve this story somehow? I don't even see the breadcrumbs displaying in the storybook.

Reply:
I spent like an hour yesterday trying to move the sidebar to give it more space, you'll have to make them smaller for now for it to show. (Are the notes about the width noticeable and clear enough?).

We'd have to switch either to Storybook v4 or change how we import the components.

Reply:
More on that:
https://github.com/storybooks/storybook/issues/993

Reply:
Oh, I completely missed the note in the README. (ohdog). I think that should be fine for now.

Reply:
btw i still need to finish moving the readme to the center section so that will be more noticeable!

Reply:
Btw, I made the notification larger on the readme